app/controllers/customers_controller.rb

```diff
@@ -358,7 +370,7 @@ def delete_admin_only_attributes(params)
 
   def send_new_password(email)
     if email.blank?
-      flash[:notice] = "Please enter the email with which you originally signed up, and we will email you a new password."
+      flash[:notice] = "Please enter the email with which you originally signed up, and we will email you a link to reset your password."
```

Comment:
Could I request that this message be moved to `config/locales/en.yml` like the others? (I realize it was hardcoded before.)